documentation/modules/operators/con-topic-operator-store.adoc

```diff
@@ -0,0 +1,117 @@
+// Module included in the following assemblies:
+//
+// assembly-using-the-topic-operator.adoc
+
+[id='ref-topic-operator-store-{context}']
+= Topic Operator topic store
+
+[role="_abstract"]
+The Topic Operator uses Kafka to store topic metadata describing topic configuration as key-value pairs.
+The _topic store_ is based on the Kafka Streams key-value mechanism, which uses Kafka topics to persist the state.
+
+Topic metadata is cached in-memory and accessed locally.
+Updates from operations applied to the local in-memory cache are persisted to a backup topic store on disk.
+The topic store is continually synchronized with updates from Kafka topics or Kubernetes `KafkaTopic` custom resources.
+Operations are handled rapidly with the topic store set up this way,
+but should the in-memory cache crash it is automatically repopulated from the persistent storage.
+
+== Internal topic store topics
+
+Internal topics support the handling of topic metadata.
+
+`__strimzi_store_topic`:: Input topic for storing the topic metadata
+`__strimzi-topic-operator-kstreams-topic-store-changelog`:: Retains a log of compacted topic store values
+
+IMPORTANT: Do not delete these topics, as they are essential to the running of the Topic Operator.
+
+== Migrating topic metadata from ZooKeeper
+
+In previous releases of Strimzi, topic metadata was stored in ZooKeeper.
```

Comment:
This information might be helpful earlier on as well.

Reply:
If we keep the mention only here, we can drop this section cleanly when it is no longer needed.